### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by jecode93**
## Pull request for the project: Hello Microverse

In this project I : 

 - Create a new GitHub repo.
 - Create a new branch according to the GitHub flow.
 - Add a .gitignore file.
 - Add node_modules to the .gitignore file.
 - Set up a linter for HTML and CSS.
 - Add descriptive README file.
 - Add 3 files : 
      - index.html 
      - styles.css 
      - test.md file "That ignores by .gitignore file".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
[the remark this answers is not in the document]
Comment (jecode93) [reply README.md]: Hello @Strangeal,

Added the sections as you suggested and thank you so much.

[the remark this answers is not in the document]
Comment (Strangeal) [reply README.md]: @jecode93 You're welcome, Kindly resubmit the project on your student dashboard for review.

[the remark this answers is not in the document]
Comment (jecode93) [reply README.md]: @Strangeal, thank you... 
Everything is ok now?

File: README.md [hunk 2]
```diff
@@ -1 +1,138 @@
...
+- CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Markup Language</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
```
Comment (topeogunleye) [inline README.md:40]: - [ ] The link to HTML here goes to React.js. I kindly suggest that you make use of the HTML documentation link instead. e.g 

`https://reactjs.org/](https://developer.mozilla.org/en-US/docs/Web/HTML
`

File: README.md [hunk 3]
```diff
@@ -1 +1,138 @@
...
+  </ul>
+</details>
+
+
+<details>
+<summary>Style</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">CSS</a></li>
```
Comment (topeogunleye) [inline README.md:48]: [OPTIONAL]
 - [ ] The link to CSS here goes to postgresql. I kindly suggest that you make use of the CSS documentation link instead. e.g

`https://developer.mozilla.org/en-US/docs/Web/CSS"`

**Review by topeogunleye — APPROVED** [review]
### Hi @jecode93, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added all relevant sections of the README template provided in the project :heavy_check_mark: 
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (jecode93) [reply README.md:40]: Hello @topeogunleye, thank you so much... 
I will do the changes and merge!

**Pull request merged**